src/Frontend.hs

```diff
@@ -60,11 +61,15 @@ runFrontend cfg = do
     persist <- Persistent.Implementation.STM.mkRunPersist
     let runAction :: Action Persistent.Implementation.STM.Persist :~> ExceptT ServantErr IO
         runAction = mkRunAction (ActionEnv persist cfg)
-        aulaTopProxy = Proxy :: Proxy AulaTop
-        stateProxy   = Proxy :: Proxy UserState
+
+        aulaTopProxy           = Proxy :: Proxy AulaTop
+        stateProxy             = Proxy :: Proxy UserState
         aulaMainOrTestingProxy = Proxy :: Proxy (AulaMain :<|> "testing" :> AulaTesting)
```

Comment:
Why not put the api next to main and testing? On the one hand one might want to use a different stack but so far this is the same so there is no need to spawn the whole thing twice (session handling and so on).